R/PipeOpFDABsignal.R

```diff
@@ -0,0 +1,105 @@
+#' @title B-spline Feature Extraction
+#' @name mlr_pipeops_fda.bsignal
+#'
+#' @description
+#' This `PipeOp` extracts features from functional data using B-spline basis functions.
```

Comment:
What are the features that are extracted? The coefficients I guess?

Reply:
yeah the basis coefficients/weights

Reply:
maybe document this